service/create/templates.go

```diff
@@ -69,24 +69,4 @@ USERDATA_FILE={{.MachineType}}
     quay.io/coreos/awscli:025a357f05242fdad6a81e8a6b520098aa65a600 -- aws s3 --region {{.Region}} cp s3://{{.S3DirURI}}/$USERDATA_FILE /var/run/coreos/temp.txt
 base64 -d /var/run/coreos/temp.txt | gunzip > /var/run/coreos/$USERDATA_FILE
 exec /usr/bin/coreos-cloudinit --from-file /var/run/coreos/$USERDATA_FILE`
-
-	createCalicoEnvFileScriptTemplate = `#!/bin/bash
```

Comment:
I think the reference from where it is actually being used has also to be removed. The build is also failing. I assume this is due to the compilation error.